manager/controllers/app/policy_compiler_interface.go

```diff
@@ -86,12 +84,12 @@ func LookupPolicyDecisions(datasetID string, policyManager connectors.PolicyMana
 
 	err = ValidatePolicyDecisionsResponse(openapiResp, PolicyManagerTaxonomy)
 	if err != nil {
-		log.Print("Error after calling ValidatePolicyDecisionsResponse:", err)
+		appContext.Log.Error().Err(err).Msg("error while validating policy manager response")
```

Comment:
Please add the datasetid and uuid

Reply:
Added datasetid, uuid is already inside.